frontend/src/pages/devices/List.tsx

```diff
@@ -153,6 +155,16 @@ export const List = () => {
         return <>{ level }%</>;
     };
 
+    const renderOnline = (entity: DeviceT) => {
+        if (entity.rawDevice.status === "ONLINE") {
+            return <p>Online</p>;
```

Comment:
Can we make the text the same size? Maybe there's a `Badge` that we can use here.